### PR TITLE
48 convention page

### PR DESCRIPTION
**Pull request by matoubongrain**

### CODE REVIEW

File: _bibliography/authorGuidelines.bib [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  title={American national standard supplement to terminology for automatic control ANSI C85.1-1963 : new and revised terms}, 
+  year={1973},
+  }
+
+    @BOOK{AutomaticControlSystemDictionary,
+  authors={ Rihard Karba and al.},
+  publisher={Springer Cham}, 
+  title={Terminological Dictionary of Automatic Control, Systems and Robotics}, 
+  year={2024},
+  doi={https://doi.org/10.1007/978-3-031-35755-8}
```
Comment (GJCRESYM) [inline _bibliography/authorGuidelines.bib:51]: Comma is missing
doi={https://doi.org/10.1007/978-3-031-35755-8},
While executing, encountered error

**Review by GJCRESYM — CHANGES_REQUESTED** [review]
image

References 5 & 7 are not pointing to titles first. Not sure it that was intended.

File: _bibliography/authorGuidelines.bib (hunk 1 above)
Comment (matoubongrain) [reply _bibliography/authorGuidelines.bib]: corrections are provided in last push.

File: pages/about/authorGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Mathematical editing
+
+please consult the {% cite EditingMathematics %} for guidelines on how to write mathematics.
+
+### Electrical and Electronics Terms, notations and conventions
+
+please consult the {% cite StandardDictionary %} for the terms to be used in Electrical and Electronics fields.
+
+For notations and conventions, please look at this [page](pages/about/notationsAndConventions.md)
```
Comment (GJCRESYM) [inline pages/about/authorGuidelines.md:75]: Link to the page is not working.
Correction required: "notation"

File: pages/about/authorGuidelines.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+For electrical and eletronic diagram, you can refer to {% cite GraphicSymbolsElectricalElectronics %} and {% cite SupplementGraphicSymbolElectricalElectronics %}
+
+For control terminology, you can check: {% cite AutomaticControlSystemDictionary %}
+
+## References
+
+{% bibliography --file authorGuidelines  %}
```
Comment (GJCRESYM) [inline pages/about/authorGuidelines.md:83]: Still the same issue with reference 7:
image

**Review by GJCRESYM — CHANGES_REQUESTED** [review]

**Review by GJCRESYM — APPROVED** [review]

**Pull request merged**